packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -136,38 +135,16 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
   }
 
   @autobind
-  private _onFocus(ev: React.FocusEvent<HTMLInputElement>): void {
-    const { inputProps } = this.props;
-
-    if (inputProps && inputProps.onFocus) {
-      inputProps.onFocus(ev);
-    }
-
-    this.setState({ isFocused: true });
-  }
-
-  @autobind
-  private _onBlur(ev: React.FocusEvent<HTMLInputElement>): void {
-    const { inputProps } = this.props;
-
-    if (inputProps && inputProps.onBlur) {
-      inputProps.onBlur(ev);
-    }
-
-    this.setState({ isFocused: false });
-  }
-
-  @autobind
-  private _onChange(ev: React.FormEvent<HTMLInputElement>) {
+  private _onClick(ev: React.FormEvent<HTMLButtonElement>) {
```

Comment:
We moved onto using the button instead of input checkbox element. So, I could no longer use the onChange handler for the input button. But, for the component itself I think it is more correct to expose the onChange event as props. As, both onClick and onChange would occur whenever the checkbox is checked/unchecked, it would semantically appropriate to give the onChange callback than the onClick callback, for the checkbox.